public/documentation/apis/presentation_api_tutorial.md

```diff
@@ -0,0 +1,195 @@
+ 
+  elem.innerHTML= ”Message from secondary display: ” + msg;
+});
+</script>
+</head>
+  <body>
+  <input type="button" onclick="showPresentation();" value=”Show Presentation”>
+  </input>
+   
+  <div id=”msg”>Message from secondary display: N/A</div>
+  </body>
+</html>
+``` 
+
+> content.html
+``` 
+<html>
+ 
+<head>
+ 
+<script>
+  window.addEventListener("message", function(event) {
+    var msg = event.data;
+   
+    var elem = document.getElementById(“output”);
+   
+    elem.innerHTML = msg;
+    event.source.postMessage("Hello from the secondary display", "*");
+  });
+</script>
+ 
+</head>
+<body>
+ 
+Message from its opener:
+ 
+<div id=”output”>N/A</div>
+</body>
+</html>
+```
+
+Follow the instructions on [how to build a Crosswalk application](https://crosswalk-project.org/#documentation/getting_started/build_an_application) to package the above two HTML files into an Android application; then follow the instructions for [running on Android](https://crosswalk-project.org/documentation/getting_started/run_on_android.html) to launch the application on a Android 4.2+ device. Make sure there is at least one available secondary display before running the application: see the Setup Wireless Display section above.
+
+Once the application is running, clicking the “Show Presentation” button will open the `content.html` page on the secondary display.
+
+You should see the following output on the primary display:
+
+> Hello from the secondary display
+
+and this output on the secondary display:
+> Hello from the opener page
+
+If there is no available wireless display, you will get an alert dialog with the message “No available display found”.
+
+For an example of a more realistic HTML5 app using a secondary display, check out the [ImageGallery](https://github.com/crosswalk-project/crosswalk-demos/tree/master/Gallery) demo. This demonstrates how to use the Presentation API to view photos on a wireless display.
+
+
+### Details of the JavaScript API ###
+
+####`presentation.requestShow Method`####
+
+`requestShow` sends a request to the user agent for to show a presentation. If a secondary display is ready for use, a new presentation window for the HTML page specified by url will be opened on the secondary display. This secondary window is opened from the current browsing context. Once the page has finished loading, the `successCallback` is invoked, taking the window object of the new page as the input parameter. Thus, the communication between the opener window and presentation window follows the approach of [HTML5 Web Messaging](http://www.w3.org/TR/webmessaging/). You can call `postMessage` to post a message and register an `onmessage` event listener to handle the message from the other side.
+
+If one of the following conditions happens, requestShow will fail and the errorCallback is called with an DOMError object. The following table describes the various possible errors:
+
+![img](pa-graphic.png)
```

Comment:
how about just creating a table inline.  Then the CSS style of the table will be consistent.